### PR TITLE
fix(weights): Fix `SessionValidatorSetMigration` weight

### LINKED ISSUE #2696: Migrations exceed block weight
### Problem to Solve

Current migrations make up 127% of block weight. Possibly next runtime upgrade will fail

### Possible Solution

Something like migration pallet with steps in `on_initialize`

### Notes

_No response_

### PR DESCRIPTION
**Pull request by ark0f**
Resolves #2696

### CODE REVIEW

**Comment by ark0f** [comment]
Now weight is just 38.75%

File: runtime/common/src/migration.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            1 // read pallet session validators
+                + 1 // read validator set
+                + 1, // read approved validator set
+        );
+
+        let session_validators = pallet_session::Pallet::<T>::validators();
+        let validator_set = validator_set::Validators::<T>::get();
+        let approved_validator_set = validator_set::ApprovedValidators::<T>::get();
+
+        if session_validators == validator_set && session_validators == approved_validator_set {
```
Comment (ukint-vs) [inline runtime/common/src/migration.rs:47]: Can you add checks that the current runtime version is under v140?

**Review by ukint-vs — APPROVED** [review]
Nice.

File: runtime/common/src/migration.rs (hunk 1 above)
Comment (ark0f) [reply runtime/common/src/migration.rs:47]: We don't need it because condition will be true for `spec_version >= 140`

Comment (ukint-vs) [reply runtime/common/src/migration.rs:47]: It is also be true after v140. We can't run this twice.

**Pull request merged**